### ISSUE TITLE
replay broken on master

### ISSUE THREAD

**Issue by BazookaJoe1900** [PX4/Firmware#8859]
posix_sitl_replay build doesn't work.
Should it?
is there manual of how to use it? I could only find data about [system_wide_replay](https://dev.px4.io/en/debug/system_wide_replay.html)

**Comment by dagar** [comment]
I believe it's done with the regular posix_sitl_default (with an environment variable) set and posix_sitl_replay can be deleted.

![image](https://user-images.githubusercontent.com/84712/36033205-5b97be60-0d7e-11e8-94b4-c4c28cf5582b.png)

I'll verify and delete.

**Comment by hamishwillee** [comment]
@dagar So [system wide replay](https://dev.px4.io/en/debug/system_wide_replay.html) is correct way to do replay right? (I'm just checking this is correct, and all you're doing is deleting the unsupported build config?)

**Comment by dagar** [comment]
I don't know the full history, but that appears to be the case. We should add something basic for replay in CI, but as an example, and to keep it working. We also need to make more of an effort to clean up as things evolve, otherwise it's very hard to follow.

**Comment by bkueng** [comment]
> So system wide replay is correct way to do replay right?

Yes.

`posix_sitl_replay` is/was used in combination with sdlog2 (which is why we still kept it, but we can start to remove it now).

**Comment by BazookaJoe1900** [comment]
This can be closed according to answers
